Doc/library/signal.rst

```diff
@@ -193,10 +193,10 @@ The :mod:`signal` module defines the following functions:
    canceled (only one alarm can be scheduled at any time).  The returned value is
    then the number of seconds before any previously set alarm was to have been
    delivered. If *time* is zero, no alarm is scheduled, and any scheduled alarm is
-   canceled.  If the return value is zero, no alarm is currently scheduled.  (See
-   the Unix man page :manpage:`alarm(2)`.)
+   canceled.  If the return value is zero, no alarm is currently scheduled.
 
-   .. availability:: Unix.
+   .. availability:: Unix (see the man page :manpage:`alarm(2)` for further
+   information).
```

Comment:
```suggestion
       information).
```

Reply:
I think I prefer it without the parenthesis, e.g `Unix. See the man page .. for further information.` Same thing for the rest.

Reply:
Updated, thank you Mariatta.